spec/features/consumer/shopping/cart_spec.rb

```diff
@@ -32,10 +32,10 @@
       it "rounds fee calculations correctly" do
         # $0.86 + 20% = $1.032
         # Fractional cents should be immediately rounded down and not carried through
-        expect(page).to have_selector '.cart-item-price',         text: '$1.03'
-        expect(page).to have_selector '.cart-item-total',         text: '$8.24'
-        expect(page).to have_selector '.order-total.item-total',  text: '$8.24'
-        expect(page).to have_selector '.order-total.grand-total', text: '$8.24'
+        expect(page).to have_selector '.cart-item-price',         text: "#{@currency_symbol}1.03"
```

Comment:
I don't understand the connection to the wait time here.

If we don't use that helper, what do we use instead? We should probably unify how we display prices and have a helper for that.